lib/web_ui/lib/src/engine/keyboard.dart

```diff
@@ -61,6 +65,16 @@ class Keyboard {
     ui.window.onPlatformMessage('flutter/keyevent',
         _messageCodec.encodeMessage(eventData), _noopCallback);
   }
+
+  bool _shouldPreventDefault(html.KeyboardEvent event) {
+    switch (event.key) {
+      case 'Tab':
+        return true;
+
+      default:
+        return false;
+    }
```

Comment:
Question: I don't know how custom shortcuts work very well. For example if a user/operating system defined a combination of keys as tab (for example F3 also works as tab), will this code still work?

Reply:
That will depend on how the OS is sending the event to the browser.

My guess is that the OS or the keyboard is doing the mapping and sending a `Tab` key event to the browser. For example, some keyboards have a `Copy` and `Paste` keys. So I attached a `keydown` event listener in the browser and clicked the `Copy` key in my keyboard. I got a `ctrl` then a `c`.